dali/kernels/alloc.h

```diff
@@ -33,7 +33,7 @@ struct Deleter {
   AllocType alloc_type;
   inline void operator()(void *p) noexcept { Deallocate(alloc_type, p, device); }
 };
-Deleter GetDeleter(AllocType type) noexcept;
+DLL_PUBLIC Deleter GetDeleter(AllocType type) noexcept;
```

Comment:
Why so many of those DLL_PUBLIC needed to be added?

Reply:
because now they have to be called from one lib to another so they need to be exported symbols